easybuild/easyconfigs/b/beagle-lib/beagle-lib-20120124-intel-2015a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'beagle-lib'
+version = '20120124'
+
+homepage = 'http://code.google.com/p/beagle-lib/'
+description = """beagle-lib is a high-performance library that can perform the core
+ calculations at the heart of most Bayesian and Maximum Likelihood
+ phylogenetics packages."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+# there is no tarball provided, only SVN checkout through:
+# svn checkout http://beagle-lib.googlecode.com/svn/trunk/ beagle-lib
+sources = [SOURCE_TGZ]
+
+dependencies = [
+    ('Java', '1.8.0_31', '', True)
+]
+builddependencies = [
+    ('Automake', '1.15', '', ('GCC', '4.9.2')),
```

Comment:
this is not enough. ALWAYS add the `autotools` bundle when you want to run `autogen.sh`.